components/asic/include/fayksic.h

```diff
@@ -0,0 +1,40 @@
+#ifndef FAYKSIC_H_
+#define FAYKSIC_H_
+
+#include "common.h"
+#include "driver/gpio.h"
+#include "job.h"
+#include <stdint.h>
+//#include "mining.h"
+
+#define ASIC_FAYKSIC_JOB_FREQUENCY_MS 2000
+#define FAYKSIC_ASIC_DIFFICULTY 256
+
+#define FAYKSIC_SERIALTX_DEBUG false
+#define FAYKSIC_SERIALRX_DEBUG false
+#define FAYKSIC_DEBUG_WORK false //causes insane amount of debug output
+#define FAYKSIC_DEBUG_JOBS false //causes insane amount of debug output
+
+
+typedef struct __attribute__((__packed__))
+{
+    uint8_t work_id;
+    uint8_t num_midstates;
+    uint8_t starting_nonce[4];
+    uint8_t nbits[4];
+    uint8_t ntime[4];
+    uint8_t merkle_root[32];
+    uint8_t prev_block_hash[32];
+    uint8_t version[4];
+} work;
+
```

Comment:
This is most likely a "more readable" approach to of "build 80 byte block header"

I can probably migrate to something like this, I just need to do some research to better understand - def more readable.